bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -289,8 +328,46 @@ spec:
         displayName: Conditions
         path: conditions
       version: v1alpha1
-    - kind: DataUpload
+    - description: DataUpload represents a data upload of a volume snapshot. There
```

Comment:
Please add the descriptions that got merged upstream: https://github.com/vmware-tanzu/velero/pull/7028/files

Reply:
Changed.